installer/Settings.wxi

```diff
@@ -129,6 +129,10 @@
   <?ifndef env.MumbleNoG15 ?>
     <?define G15 = true ?> 
   <?endif ?>
+  
+  <?ifndef env.MumbleNoRNNoise ?>
+    <?define RNN = true ?>
```

Comment:
This looks wrong. The env is `no rnnoise` but then RNN is set to true, which makes the dll be included by the ifdet??